src/SteamAPI.ts

```diff
@@ -336,23 +336,23 @@ export default class SteamAPI {
 	async getGameDetails(
 		app: number | number[],
 		{ language = this.language, currency = this.currency, filters = [] } = {}
-	): Promise<GameDetails | { [key: number]: GameDetails } | undefined> {
```

Comment:
I'm all for this change, just noting this will be a breaking change as current implementations using the previous version could expect to return details directly when given a single app ID.